packages/html-to-structured-text/src/handlers.ts

```diff
@@ -359,6 +359,14 @@ export const span: Handler<HastTextNode> = async function span(
   });
 };
 
+export const newLine: Handler<HastTextNode> = async function newLine(
+  createNode,
+) {
+  return createNode('span', {
```

Comment:
If new lines are allowed anywhere in Dast, then this is fine!

With this, top level `br` will be transformed to a paragraph with a new line. Probably it is fine